system-tests/tests/smoke/cre/cmd/environment/environment.go

```diff
@@ -0,0 +1,345 @@
+package environment
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/environment/devenv"
+	cldlogger "github.com/smartcontractkit/chainlink/deployment/logger"
+	libcontracts "github.com/smartcontractkit/chainlink/system-tests/lib/cre/contracts"
+
+	chainselectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/framework"
+	"github.com/smartcontractkit/chainlink-testing-framework/framework/components/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/framework/components/jd"
+	ns "github.com/smartcontractkit/chainlink-testing-framework/framework/components/simple_node_set"
+	"github.com/smartcontractkit/chainlink-testing-framework/seth"
+	libc "github.com/smartcontractkit/chainlink/system-tests/lib/conversions"
+	crecontracts "github.com/smartcontractkit/chainlink/system-tests/lib/cre/contracts"
+	"github.com/smartcontractkit/chainlink/system-tests/lib/cre/don/jobs/chainreader"
+	crepor "github.com/smartcontractkit/chainlink/system-tests/lib/cre/don/jobs/por"
+	"github.com/smartcontractkit/chainlink/system-tests/lib/cre/don/jobs/webapi"
+	creenv "github.com/smartcontractkit/chainlink/system-tests/lib/cre/environment"
+	cretypes "github.com/smartcontractkit/chainlink/system-tests/lib/cre/types"
+	libtypes "github.com/smartcontractkit/chainlink/system-tests/lib/types"
+)
+
+var EnvironmentCmd = &cobra.Command{
+	Use:   "env",
+	Short: "Environment commands",
+	Long:  `Commands to manage the environment`,
+}
+
+func init() {
+	EnvironmentCmd.AddCommand(startCmd)
+	EnvironmentCmd.AddCommand(DeployAndSetupKeystoneConsumerCmd)
+
+	// add flags to the command
+	DeployAndSetupKeystoneConsumerCmd.Flags().StringVar(&rpcHTTPURL, "rpc-http-url", "", "RPC HTTP URL")
```

Comment:
Are we separating Keystone consumer from TOML config on purpose? There is another way to configure it through TOML overrides via CTF_CONFIGS

Reply:
no way, I might remove it soon. I used it to deploy consumer so that I could validate that env works